api/src/models/biohub-create.ts

```diff
@@ -0,0 +1,74 @@
+import { ObservationRecord } from '../repositories/observation-repository';
+import { getLogger } from '../utils/logger';
+import { GetSurveyData } from './survey-view';
+
+const defaultLog = getLogger('models/biohub-create');
+
+/**
+ * Object to be sent to Biohub API for creating an observation.
+ *
+ * @export
+ * @class PostSurveyObservationToBiohubObject
+ */
+export class PostSurveyObservationToBiohubObject {
+  id: string;
+  type: string;
+  properties: object;
+  // features: ObservationRecord[];
+
+  constructor(obj: ObservationRecord) {
```

Comment:
nit: doesn't really matter, but since `obj` has a defined type, you could rename it to something like `observationRecord`, just for clarity. Same with the classes below.